### PR TITLE
Feature/id 41/Add Checkout Modal

### PR DESCRIPTION
**Pull request by gabitrujillo**

### CODE REVIEW

**Review by victor-fundacion — APPROVED** [review]

**Review by victor-fundacion — CHANGES_REQUESTED** [review]
Fix, CI errors

**Review by victor-fundacion — APPROVED** [review]

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+export default function MyEventsPage() {
+  const searchParams = useSearchParams();
+  const eventName = searchParams.get('eventName') || 'Unnamed Event';
+  const quantity = searchParams.get('quantity') || '0';
+  const pricePerTicket = searchParams.get('pricePerTicket') || '0';
+  const totalPrice = parseFloat(searchParams.get('totalPrice') || '0');
+  const coverPhotoUrl = searchParams.get('coverPhotoUrl'); // Imagen del evento
+
+  // Calculate platform fee (3% of total price)
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  const platformFee = totalPrice * 0.03;
+
+  // Calculate final total price after adding the platform fee
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+import checkoutStyles from '@/styles/components/CheckoutStyles';
+import { EventDetailDto } from '@/utils/interfaces/EventInterfaces';
+
+export default function MyEventsPage() {
+  const searchParams = useSearchParams();
+  const eventName = searchParams.get('eventName') || 'Unnamed Event';
+  const quantity = searchParams.get('quantity') || '0';
+  const pricePerTicket = searchParams.get('pricePerTicket') || '0';
+  const totalPrice = parseFloat(searchParams.get('totalPrice') || '0');
+  const coverPhotoUrl = searchParams.get('coverPhotoUrl'); // Imagen del evento
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+  // Calculate final total price after adding the platform fee
+  const finalTotalPrice = totalPrice + platformFee;
+
+  return (
+    <Box sx={checkoutStyles.container}>
+      <Typography sx={checkoutStyles.title}>Checkout</Typography>
+
+      <Grid container spacing={4}>
+        {/* Left Section: Payment Form */}
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+          </Paper>
+        </Grid>
+
+        {/* Right Section: Order Summary */}
+        <Grid item xs={12} md={5}>
+          <Paper elevation={3} sx={checkoutStyles.orderSummary}>
+            {/* Imagen del evento */}
+            {coverPhotoUrl && (
+              <img
+                src={coverPhotoUrl} // Imagen dinámica
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+              variant="contained"
+              sx={checkoutStyles.payNowButton}
+              fullWidth
+            >
+              Pay Now
+            </Button>
+          </Paper>
+        </Grid>
+
+        {/* Right Section: Order Summary */}
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+        <Grid item xs={12} md={5}>
+          <Paper elevation={3} sx={checkoutStyles.orderSummary}>
+            {/* Imagen del evento */}
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+            {coverPhotoUrl && (
+              <img
+                src={coverPhotoUrl} // Imagen dinámica
+                alt="Event Cover"
+                style={{
+                  width: '100%', // Ancho completo
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+                  height: 'auto', // Mantiene la proporción
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+                  borderRadius: '8px', // Bordes redondeados
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+                  marginBottom: '16px', // Espacio debajo de la imagen
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

File: apps/cevent_frontend/src/app/checkout/page.tsx [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+                }}
+              />
+            )}
+
+            {/* Resumen de la orden */}
```
Comment (Karvlox) [inline apps/cevent_frontend/src/app/checkout/page.tsx]: Delete comment.

**Review by Karvlox — APPROVED** [review]

**Pull request merged**